tests/admin_views/tests.py

```diff
@@ -6275,6 +6276,152 @@ def test_hidden_fields_small_window(self):
         finally:
             self.selenium.set_window_size(current_size["width"], current_size["height"])
 
+    def test_updating_related_objects_updates_fk_selects(self):
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.support.ui import Select
+
+        BORN_COUNTRY_SELECT_ID = "id_born_country"
+        LIVING_COUNTRY_SELECT_ID = "id_living_country"
+        FAVORITE_COUNTRY_TO_VACATION_SELECT_ID = "id_favorite_country_to_vacation"
+        CONTINENT_SELECT_ID = "id_continent"
+
+        def _get_HTML_inside_element_by_id(id_):
+            return self.selenium.find_element(By.ID, id_).get_attribute("innerHTML")
+
+        self.admin_login(
+            username="super", password="secret", login_url=reverse("admin:index")
+        )
+        add_url = reverse("admin:admin_views_traveler_add")
+        self.selenium.get(self.live_server_url + add_url)
+
+        # Add new Country from the BORN select
+        self.selenium.find_element(By.ID, f"add_{BORN_COUNTRY_SELECT_ID}").click()
+        self.wait_for_and_switch_to_popup()
+        self.selenium.find_element(By.ID, "id_name").send_keys("Argentina")
+        continent_select = Select(
+            self.selenium.find_element(By.ID, CONTINENT_SELECT_ID)
+        )
+        continent_select.select_by_visible_text("South America")
+        self.selenium.find_element(By.CSS_SELECTOR, '[type="submit"]').click()
+        self.selenium.switch_to.window(self.selenium.window_handles[0])
+
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(BORN_COUNTRY_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            <option value="1" selected="">Argentina</option>
+            """,
+        )
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(LIVING_COUNTRY_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            <option value="1">Argentina</option>
+            """,
+        )
+        # Argentina won't appear because `favorite_country_to_vacation`
+        # field has `limit_choices_to`
+        self.assertHTMLEqual(
+            _get_HTML_inside_element_by_id(FAVORITE_COUNTRY_TO_VACATION_SELECT_ID),
+            """
+            <option value="" selected="">---------</option>
+            """,
```

Comment:
This can be single-lined.
```suggestion
            '<option value="" selected="">---------</option>',
```